### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by masylum**
This commit tries to clean some of the code. I've removed configurations
that are currently not used and may be confusing to people (such as the
`lv-lspinstall` and the `lv-spectre`).

I also took the chance to clean a bit the indentation which was a bit
random.

### CODE REVIEW

File: init.lua [hunk 1]
```diff
@@ -3,7 +3,6 @@ require('keymappings')
 vim.cmd('luafile ' .. CONFIG_PATH .. '/lv-config.lua')
 require('settings')
 require('plugins')
-require('lv-utils')
```
Comment (masylum) [inline init.lua:6]: this is not needed since it just exposes a function

File: lua/default-config.lua [hunk 2]
```diff
@@ -82,7 +75,7 @@ O = {
             autoformat = false,
             isort = false,
             diagnostics = {
-                virtual_text = {spacing = 0, prefix = ""},
+                virtual_text = {spacing = 0, prefix = "⎨"},
```
Comment (masylum) [inline lua/default-config.lua:78]: ooops, this was not intended to be in this commit... I can remove it if you prefer the previous default

File: lua/default-config.lua [hunk 3]
```diff
@@ -191,10 +184,10 @@ O = {
             },
             autoformat = false,
             diagnostics = {
-                virtual_text = {spacing = 0, prefix = ""},
+                virtual_text = {spacing = 0, prefix = "⎨"},
```
Comment (masylum) [inline lua/default-config.lua:187]: should we DRY all those diagnostics configuration?

**Comment by IndianBoy42** [comment]
Was the indentation done with a formatter? eventually it should as thats the only practical way to get consistency, but I think the formatting should be separated from the rest of the changes in this PR, because the 'official' formatter (stylua/lua-format/lua-fmt) hasn't been chosen yet

**Comment by masylum** [comment]
I've just used `=` to force the 4 spaces instead of 2. I can split it into smaller PRs if need be.

**Comment by luxus** [comment]
there is some "hidden" discussion about cleaning the Lua files in https://github.com/ChristianChiarulli/LunarVim/issues/652

**Comment by luxus** [comment]
@masylum you should run stylua script in extras on your branch. The changed shouldn't be that much after that

**Pull request closed without merging**